plugins/module_utils/network/sonic/config/bgp_neighbors/bgp_neighbors.py

```diff
@@ -326,19 +326,19 @@ def build_bgp_peer_groups_payload(self, cmd, have, bgp_as, vrf_name):
                     peer_group_cfg.update({'peer-group-name': peer_group['name']})
                     bgp_peer_group.update({'peer-group-name': peer_group['name']})
                 if peer_group.get('bfd', None) is not None:
-                    bgp_peer_group.update({'openconfig-bfd:enable-bfd': {'config': {'enabled': peer_group['bfd']}}})
+                    bgp_peer_group.update({'enable-bfd': {'config': {'enabled': peer_group['bfd']}}})
```

Comment:
This change is a little different from the usual OC Yang compliance changes (because it is not related to an "ext" reference. Is this also related to an OC Yang change?